### PR TITLE
Structure tutorial next-steps

### PR DESCRIPTION
**Pull request by imbstack**
This was motivated by me wanting the links at the bottom of the pages to be styled a bit differently, but I think the new way the links are defined is a pretty nice side-effect of this. I'm 100% up for changing some bit of how this works/looks.

Right now the links are on the [bottom of the page where they were before](https://imbstack.keybase.pub/bottom-next-steps.mov), but perhaps [the sidebar](https://imbstack.keybase.pub/sidebar-next-steps.mov) is a better place for them?

### CODE REVIEW

File: src/tutorial/gecko-tasks.md [hunk 1]
```diff
@@ -15,8 +23,3 @@ When a developer pushes to a Gecko repository, a long chain of events begins:
 Due to its "self-service" desgn, very little of this process is actually part of TaskCluster, so we provide only a brief overview and some pointers.
 TaskCluster provides a small bit of glue ([mozilla-taskcluster](/manual/vcs/mozilla-taskcluster)) to create the decision task, but the task itself is defined entirely in-tree.
 Similarly, TreeHerder is a separate service, not operated as a part of TaskCluster, although mozilla-taskcluster does feed data to it based on `task.extra`.
-
- * [How is the decision task defined?](gecko-decision-task) - TODO
- * [How is the task-graph generated?](gecko-task-graph) - TODO
- * [How do try pushes work?](gecko-try-pushes) - TODO
- * [How are the docker images created?](gecko-docker-images) - TODO
```
Comment (djmitche) [inline src/tutorial/gecko-tasks.md]: I could have sworn I did these..

Comment (djmitche) [inline src/tutorial/gecko-tasks.md]: Oh, I did, that's #111

**Comment by djmitche** [comment]
I like this a lot.  I'm sort of torn regarding sidebar or bottom.  The advantage of the bottom is, it's right there when you finish reading the page.  The disadvantage is, if you're trying to navigate to something, it is annoying to have to scroll down on each page.  I suppose using both would be crazy?

File: src/tutorial/gecko-tasks.md (hunk 1 above)
Comment (imbstack) [inline src/tutorial/gecko-tasks.md]: If we merge #111 now, I'm happy to rebase on top of it.

**Comment by imbstack** [comment]
Yeah, I don't know which one I like more either. Both probably is a bit crazy. On account of my laziness I vote that we stick with the bottom for now since I don't have to change anything to do that? It's just a couple of lines to move to the sidebar later if we want.

**Comment by djmitche** [comment]
+1 to that :)

**Pull request merged**